dashboard.php

```diff
@@ -0,0 +1,58 @@
+<?php
+$db = new PDO('mysql:dbname=portfolio;host=127.0.0.1', 'root');
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+$stmt = $db->query("SELECT `title` ,`intro_text` FROM `intro`");
+$title_array = $stmt->fetch();
+$stmt = $db->query("SELECT `paragraph_1`, `image`, `paragraph_2` FROM `about_me`");
+$about_me_array = $stmt->fetch();
+$stmt = $db->query("SELECT `title`, `image`, `paragraph` FROM `projects`");
+$projects_array = $stmt->fetch();
+
+$data = array_merge($title_array, $about_me_array, $projects_array);
+?>
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link href="https://fonts.googleapis.com/css?family=Quicksand" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Fanwood+Text" rel="stylesheet">
+    <link rel="stylesheet" type="text/css" href="css/normalize.css">
+    <link rel="stylesheet" type="text/css" href="css/portfolio.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Dashboard</title>
+</head>
+<body>
+    <h1>DASHBOARD PAGE</h1>
+    <a class="nav-text" href="index.php">Home Page</a>
+        <form action="upload_intro.php" method="post" enctype="multipart/form-data">
+                <h3>Intro:</h3>
+                    Title text:
+                        <input name="title_text" input type="text" value="<?php echo $title_array["title"] ?>"><br>
+                    Intro text:
+                        <textarea name="intro_text" input type="text" rows="10" cols="40"><?php echo $title_array["intro_text"] ?></textarea><br>
+                        <input type="submit" name="Submit All" value="submit all">
+            </form>
```

Comment:
check indenting for different forms and inputs etc

Reply:
done. good shout.